### PR TITLE
fix: fix mime types with multiple extensions (Issue #397)

### PR DESCRIPTION
**Pull request by mikitabut**
#397

### CODE REVIEW

File: src/components/Files/FileManagerModal.tsx [hunk 1]
```diff
@@ -354,7 +354,9 @@ export const FileManagerModal = ({
                   {t(
                     'Max file size up to 512 Mb. Supported types: {{allowedExtensions}}.',
                     {
-                      allowedExtensions: allowedExtensions.join(', '),
+                      allowedExtensions:
+                        allowedExtensions.join(', ') ||
+                        'no available extensions',
```
Comment (IlyaBondar) [inline src/components/Files/FileManagerModal.tsx:359]: Maybe should be ` all` by default?

**Review by IlyaBondar — APPROVED** [review]

**Review by denys-kolomiitsev — APPROVED** [review]

**Comment by IlyaBondar** [comment]
are all extensions now available for `image/jpeg`?
image

File: src/components/Files/FileManagerModal.tsx (hunk 1 above)
Comment (mikitabut) [reply src/components/Files/FileManagerModal.tsx:359]: For all types we have special `*/*` case, so this is not real case, only when incorrect configuration

**Comment by mikitabut** [comment]
> 

Yes

**Pull request merged**